### DISCUSSION TITLE
Same Patient launching issue

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#543] [General]
Hi @nroduit, if I try to launch the same patient, it doesn't launch and focuses here. Header is coming empty
Here is the log

5.04.2024 17:54:36.993 *INFO* [pipe-dicom:rs --url "http://xxxxxx:8080/dcm4chee-arc/aets/xxxxx/rs" -r "studyUID=1.3.1xx2.2.1107.5.8.15.100372.30000018061909240398100000315" --query-ext "&includedefaults=false" IID_URL_TARGET=_self] org.weasis.dicom.explorer.DicomModel: PRIOR PACS URL:=http://xxxxx:8080/dcm4chee-arc/aets/xxxx-QA/rs
15.04.2024 17:54:36.993 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://xxxxx:8080/dcm4chee-arc/aets/xxxx/rs/studies?0020000D=1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315&includefield=00080020,00080030,00080050,00080061,00080090,00081030,00100010,00100020,00100021,00100030,00100040,0020000D,00200010&includedefaults=false
15.04.2024 17:54:36.994 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:54:37.050 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://xxxx:8080/dcm4chee-arc/aets/xxxx-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series?includefield=0008103E,00080060,0020000E,00200011,00081190&includedefaults=false
15.04.2024 17:54:37.051 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:54:37.130 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: Adding new patient: CONFIDENTIAL
15.04.2024 17:54:37.131 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://xxxxx:8080/dcm4chee-arc/aets/xxxx-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:54:37.131 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:54:37.206 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: loadThumbnail() instance and wadoParameters :=org.weasis.dicom.mf.SopInstance@9a54ba01org.weasis.dicom.mf.WadoParameters@6ca603ec
15.04.2024 17:54:37.206 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: URLParameters params :=org.weasis.core.api.util.URLParameters@1f3d13c8
15.04.2024 17:54:37.206 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getThumbURL() thumbURL :=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52
15.04.2024 17:54:37.208 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getThumbURL() URL :=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/thumbnail?viewport=256%2C256
15.04.2024 17:54:37.208 *INFO* [AWT-EventQueue-0] org.weasis.dicom.explorer.DicomExplorer: Add series: Exported MR Basic Reading
15.04.2024 17:54:37.208 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: headers  headers :={Accept=multipart/related;type="application/dicom";transfer-syntax=*}
15.04.2024 17:54:37.209 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: headers  Accept :={Accept=image/jpeg}
15.04.2024 17:54:37.209 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: URLParameters paramsorg.weasis.core.api.util.URLParameters@3510d704
15.04.2024 17:54:37.209 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={Accept=image/jpeg}
15.04.2024 17:54:37.283 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getResponseCode() code200
15.04.2024 17:54:37.288 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: outFile fileC:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\cache\thumb\thumb_14513903380160673784.jpg
15.04.2024 17:54:37.288 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Downloading studies 13-Feb-2024 - Pathology_L-Spine_Brochure!?
15.04.2024 17:54:37.288 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Downloading series of CONFIDENTIAL [Exported MR Basic Reading]
15.04.2024 17:54:37.288 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getWadoBaseURL() baseURL
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getDirectDownloadFile() requesthttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getAdditionnalParameters() requesthttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getUrl() urlhttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.290 *DEBUG* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Download DICOM instance http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12 index 0.
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getWadoBaseURL() baseURL
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getDirectDownloadFile() requesthttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.290 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getAdditionnalParameters() requesthttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.291 *INFO* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getUrl() urlhttp://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.291 *DEBUG* [Series Downloader-pool-5-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Download DICOM instance http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13 index 1.
15.04.2024 17:54:37.293 *DEBUG* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: prior file exists check exists:=false for C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\downloading\1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12.dcm.
15.04.2024 17:54:37.293 *DEBUG* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: prior file exists check exists:=false for C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\downloading\1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13.dcm.
15.04.2024 17:54:37.293 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={Accept=multipart/related;type="application/dicom";transfer-syntax=*}
15.04.2024 17:54:37.294 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={Accept=multipart/related;type="application/dicom";transfer-syntax=*}
15.04.2024 17:54:37.319 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getHttpResponse() url :=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.320 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: urlcon := org.weasis.core.api.util.ClosableURLConnection@189617f
15.04.2024 17:54:37.320 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getResponseCode() code := 200
15.04.2024 17:54:37.320 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: urlcon getInputStream():sun.net.www.protocol.http.HttpURLConnection$HttpInputStream@f4e8f3e
15.04.2024 17:54:37.322 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: createTempFile tempFile :=C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\downloading\image_5721981538111038220.dcm
15.04.2024 17:54:37.322 *DEBUG* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Start to download DICOM instance http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12 to image_5721981538111038220.dcm.
15.04.2024 17:54:37.322 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getOverrideList() overrideList := null
15.04.2024 17:54:37.323 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: multipart handler := org.weasis.dicom.explorer.wado.LoadSeries$Download$$Lambda$537/0x00000008005c75c8@4e738505
15.04.2024 17:54:37.333 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getHttpResponse() url :=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.333 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: urlcon := org.weasis.core.api.util.ClosableURLConnection@335a27d2
15.04.2024 17:54:37.333 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getResponseCode() code := 200
15.04.2024 17:54:37.333 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: urlcon getInputStream():sun.net.www.protocol.http.HttpURLConnection$HttpInputStream@3d9ddfc9
15.04.2024 17:54:37.335 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: createTempFile tempFile :=C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\downloading\image_4246924390793807649.dcm
15.04.2024 17:54:37.335 *DEBUG* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: Start to download DICOM instance http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13 to image_4246924390793807649.dcm.
15.04.2024 17:54:37.335 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getOverrideList() overrideList := null
15.04.2024 17:54:37.336 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: multipart handler := org.weasis.dicom.explorer.wado.LoadSeries$Download$$Lambda$537/0x00000008005c75c8@39a24ca2
15.04.2024 17:54:37.367 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: bytesTransferred readBytes := -1
15.04.2024 17:54:37.367 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: downloadInFileCache() bytesTransferred: -1
15.04.2024 17:54:37.368 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: End of downloading http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12 
15.04.2024 17:54:37.368 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: renameFile renameFile:=C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\dicom\image_5721981538111038220.dcm
15.04.2024 17:54:37.368 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: printing url:=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.372 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getDicomReader() reader :=org.weasis.dicom.codec.DicomMediaIO@43b972d
15.04.2024 17:54:37.375 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getParent() patient :=CONFIDENTIAL
15.04.2024 17:54:37.375 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getParent() study :=13-Feb-2024 - Pathology_L-Spine_Brochure!?
15.04.2024 17:54:37.377 *INFO* [Image Downloader-pool-33-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getParent() dicomSeries :=Exported MR Basic Reading
15.04.2024 17:54:37.391 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: bytesTransferred readBytes := -1
15.04.2024 17:54:37.392 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: downloadInFileCache() bytesTransferred: -1
15.04.2024 17:54:37.392 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: End of downloading http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13 
15.04.2024 17:54:37.392 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: renameFile renameFile:=C:\Users\354883\AppData\Local\Temp\weasis-354883.19744F24\dicom\image_4246924390793807649.dcm
15.04.2024 17:54:37.392 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: printing url:=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315/series/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52/instances/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.397 *INFO* [Image Downloader-pool-34-thread-1] org.weasis.dicom.explorer.wado.LoadSeries: getDicomReader() reader :=org.weasis.dicom.codec.DicomMediaIO@7ca0d5f2
15.04.2024 17:54:37.404 *INFO* [AWT-EventQueue-0] org.weasis.dicom.explorer.DicomExplorer: Prior Study root selected.. Skipping the load...
15.04.2024 17:54:37.406 *INFO* [pool-35-thread-1] org.weasis.dicom.explorer.prior.PriorStudyLoader: Prior Studies PACS URL List:=[http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs]
15.04.2024 17:54:37.406 *INFO* [pool-35-thread-1] org.weasis.dicom.explorer.prior.PriorStudyLoader: Prior Studies PACS API:=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies?00100020=17540000909158&includefield=00080020,00080030,00080050,00080061,00080090,00081030,00100010,00100020,00100021,00100030,00100040,0020000D,00200010&includedefaults=false
15.04.2024 17:54:37.421 *DEBUG* [AWT-EventQueue-0] org.weasis.core.api.media.data.ImageElement: Asking for reading image: 1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.421 *DEBUG* [Image Loader-pool-10-thread-1] org.weasis.dicom.codec.DicomMediaIO: Start reading dicom image frame: 0 sopUID: 1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.12
15.04.2024 17:54:37.437 *INFO* [pool-35-thread-1] org.weasis.dicom.explorer.rs.RsQueryParams: PRIOR Response code for http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies?00100020=17540000909158&includefield=00080020,00080030,00080050,00080061,00080090,00081030,00100010,00100020,00100021,00100030,00100040,0020000D,00200010&includedefaults=false is 200
15.04.2024 17:54:37.438 *INFO* [pool-35-thread-1] org.weasis.dicom.explorer.prior.PriorStudyLoader: Prior Study Response for http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies?00100020=17540000909158&includefield=00080020,00080030,00080050,00080061,00080090,00081030,00100010,00100020,00100021,00100030,00100040,0020000D,00200010&includedefaults=false => [{"00080020":{"vr":"DA","Value":["20240213"]},"00080030":{"vr":"TM","Value":["093808.924771"]},"00080050":{"vr":"SH","Value":["355784678901235"]},"00080061":{"vr":"CS","Value":["CT"]},"00080090":{"vr":"PN"},"00081030":{"vr":"LO","Value":["Pathology_L-Spine_Brochure!?"]},"00081190":{"vr":"UR","Value":["http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315"]},"00100010":{"vr":"PN","Value":[{"Alphabetic":"Confidential"}]},"00100020":{"vr":"LO","Value":["17540000909158"]},"00100021":{"vr":"LO","Value":["ATHMA"]},"00100030":{"vr":"DA","Value":["19780726"]},"00100040":{"vr":"CS","Value":["F"]},"0020000D":{"vr":"UI","Value":["1.3.12.2.1107.5.8.15.100372.30000018061909240398100000315"]},"00200010":{"vr":"SH"}}]
15.04.2024 17:54:37.440 *DEBUG* [pool-35-thread-1] org.weasis.dicom.explorer.prior.PriorStudyLoader: priorPatientStudiesMap:={1002233431325=[16-Feb-2024 10:00 - Pathology_L-Spine_Brochure!?], 10020001298243=[15-Feb-2024 20:07 - ABDOMEN PLAIN], 17540000909158=[13-Feb-2024 09:38 - Pathology_L-Spine_Brochure!?]}
15.04.2024 17:54:37.441 *INFO* [pool-35-thread-1] org.weasis.dicom.explorer.DicomExplorer: Prior Study root selected.. Skipping the load...
15.04.2024 17:54:37.529 *INFO* [AWT-EventQueue-0] org.weasis.dicom.explorer.wado.LoadSeries: *PERF* type:DICOMDIR seriesUID:1.3.12.2.1107.5.8.15.100372.30000018061909240398100000255.52 modality:CT nbImages:2 size:205122 time:237 rate:865.49 errors:0
15.04.2024 17:54:37.667 *DEBUG* [Thread-5] org.weasis.core.api.media.data.ImageElement: Asking for reading image: 1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.667 *DEBUG* [Image Loader-pool-10-thread-1] org.weasis.dicom.codec.DicomMediaIO: Start reading dicom image frame: 0 sopUID: 1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13
15.04.2024 17:54:37.676 *DEBUG* [Thread-5] org.weasis.dicom.codec.DicomSeries: Reading time: 9 ms of image: 1.3.12.2.1107.5.8.15.100372.30000018061909240398100000024.1.13

test same patient

15.04.2024 17:58:03.854 *INFO* [pipe-dicom:rs --url "http://xxxxxx:8080/dcm4chee-arc/aets/xxxx-QA/rs" -r "studyUID=1.3.46.670589.33.1.63843624102328272700001.4954571449547756346" --query-ext "&includedefaults=false" IID_URL_TARGET=_self] org.weasis.dicom.explorer.DicomModel: PRIOR PACS URL:=http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs
15.04.2024 17:58:03.855 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://xxxxx:8080/dcm4chee-arc/aets/xxxxx-QA/rs/studies?0020000D=1.3.46.670589.33.1.63843624102328272700001.4954571449547756346&includefield=00080020,00080030,00080050,00080061,00080090,00081030,00100010,00100020,00100021,00100030,00100040,0020000D,00200010&includedefaults=false
15.04.2024 17:58:03.855 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:03.909 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series?includefield=0008103E,00080060,0020000E,00200011,00081190&includedefaults=false
15.04.2024 17:58:03.909 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:03.954 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://xxx:8080/dcm4chee-arc/aets/xxx-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624455491472500001.5459674196636167352/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:03.955 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:03.982 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624576363385900001.5705670183051498293/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:03.983 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:04.083 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624576363385900002.5111105027171513326/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:04.083 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:04.245 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624576363385900003.5469232219554687939/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:04.245 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:04.300 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624576363385900005.5471214307124818588/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:04.301 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}
15.04.2024 17:58:04.359 *DEBUG* [Dicom Model-pool-3-thread-1] org.weasis.dicom.explorer.rs.RsQueryResult: QIDO-RS request: http://azsilasqa3212:8080/dcm4chee-arc/aets/ATHMAPACS-QA/rs/studies/1.3.46.670589.33.1.63843624102328272700001.4954571449547756346/series/1.3.46.670589.33.1.63843624811696846200001.4711894320780738648/instances?includefield=00080018,00200013,00081190&includedefaults=false
15.04.2024 17:58:04.360 *INFO* [Dicom Model-pool-3-thread-1] org.weasis.core.api.util.NetworkUtil: getUnmodifiableHeaders() headers :={}

Atleast it should focus the same patient

**Comment by nroduit** [comment]
I don't understand the question. Please describe the problem precisely.
